README.md

```diff
@@ -0,0 +1,70 @@
+# Quarkus Demo Project: Order Service
+
+This project uses Quarkus, the Supersonic Subatomic Java Framework.
+
+If you want to learn more about Quarkus, please visit its website: https://quarkus.io/ .
```

Comment:
I think we don't add this basic stuff and also the parts about packaging and creating the executable. Just some sentences on what the application is about and what features it provides, as Malte mentioned in todays meeting